Classes/Bookmark/BookmarksViewController.swift

```diff
@@ -0,0 +1,220 @@
+//
+//  BookmarksViewController.swift
+//  Freetime
+//
+//  Created by Rizwan on 18/10/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+class BookmarksViewController:
+    UITableViewController,
+    UISearchBarDelegate,
+    PrimaryViewController,
+TabNavRootViewControllerType {
+  
+    private let client: GithubClient
+    private let cellIdentifier = "bookmark_cell"
+    private let bookmarkStore = BookmarksStore.shared
+    private let searchBar = UISearchBar()
+    private var filterdBookmarks: [BookmarkModel]? = nil
+    
+    // MARK: Init
+    
+    init(client: GithubClient) {
+        self.client = client
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        searchBar.delegate = self
+        searchBar.placeholder = NSLocalizedString(Constants.Strings.search, comment: "")
+        searchBar.tintColor = Styles.Colors.Blue.medium.color
+        searchBar.backgroundColor = .clear
+        searchBar.searchBarStyle = .minimal
+        searchBar.sizeToFit()
+        tableView.tableHeaderView = searchBar
```

Comment:
We'll probably move to iOS 11 nav bars, so might want to play w/ iOS 11 and `navigationItem.searchController`